Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM elasticsearch:6.8.11
+FROM elasticsearch:7.9.0
```

Comment:
I think the version we have in our cluster is 7.7, could you take an image with the same version please

Reply:
I did this on the last commit. DB dump is version specific so I had to redo it and I had only the code for the last mapping